components/session.py

```diff
@@ -4,19 +4,20 @@
 from discord.ext import tasks
 from typing import Optional
 
-from components.bot import RecruitBot
 from components.views import SessionView
 
 
 class Session:
-    bot: RecruitBot
+    # bot: RecruitBot

```

Comment:
delete commented code

Reply:
No strong feelings; I just commented everything that I needed to get rid of to fix the circular imports in case I found a better way to deal with it later